### PR TITLE
Added yolov8/7/6/x models.

### PR DESCRIPTION
**Pull request by kadirnar**

### CODE REVIEW

File: tasks/src/object-detection/data.ts [hunk 1]
```diff
@@ -60,6 +60,22 @@ const taskData: TaskDataCustom = {
 			description: "An application that contains various object detection models to try from.",
 			id:          "Gradio-Blocks/Object-Detection-With-DETR-and-YOLOS",
 		},
+		{
+			description: "Ultralytics YOLOv8: State-of-the-Art YOLO Models.",
```
Comment (merveenoyan) [inline tasks/src/object-detection/data.ts:64]: Is it okay if you could pick two of these Spaces instead, given most of them are similar? Maybe pick most distinct ones?

**Review by merveenoyan — COMMENTED** [review]
It's very elaborate how you put the descriptions, thanks a lot! Left a small comment 🙂

File: tasks/src/object-detection/data.ts (hunk 1 above)
Comment (kadirnar) [reply tasks/src/object-detection/data.ts:64]: The most recent YOLO repositories are YOLOv8 and YOLOv6.

Comment (kadirnar) [reply tasks/src/object-detection/data.ts:64]: I fixed it.

**Review by merveenoyan — APPROVED** [review]
LGTM!

**Pull request merged**